spec/models/user_spec.rb

```diff
@@ -15,6 +15,23 @@
         end
       end
     end
+
+    context "when the user is set as someone else's fnok" do
+      let(:fnok) { create(:fannish_next_of_kin) }
+      let(:user) { fnok.kin }
+      let(:person) { fnok.user }
+
+      it "explictly removes the relationship" do
```

Comment:
Typo -- there's another `i` in `explicitly` -- but let's be a little more descriptive and say something like `removes the relationship and creates a log item of the removal`.